templates/docs/examples/patterns/buttons/alignment.html

```diff
@@ -0,0 +1,30 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Buttons / Alignment{% endblock %}
+
+{% block standalone_css %}patterns_buttons{% endblock %}
+
+{% block content %}
+<p>
+  <button>Classless button</button>
+  <button>Classless button</button>
+</p>
+
+<p>
+  <button class="p-button--neutral">Neutral button</button>
+  <button class="p-button--positive">Positive button</button>
+  <button class="p-button--negative">Negative button</button>
+</p>
+
+<p>
+  <a href="#" class="p-button--neutral">Neutral link</a>
+  <a href="#" class="p-button--positive">Positive link</a>
+  <a href="#" class="p-button--negative">Negative link</a>
```

Comment:
It would be nice to have some mixed examples as that was the issue

Reply:
The block below this one has a mix of links and buttons.